packages/dds/task-manager/src/interfaces.ts

```diff
@@ -19,11 +19,19 @@ export interface ITaskManagerEvents extends ISharedObjectEvents {
 
 export interface ITaskManager extends ISharedObject<ITaskManagerEvents> {
     /**
-     * Try to lock the task.  Promise resolves when the lock is acquired, or rejects if we are removed from the
-     * queue without acquiring the lock for any reason.
+     * Volunteer for the task.  Promise resolves true when the task is assigned to the local client. It rejects if the
```

Comment:
Presently, clients should always verify whether the task is already completed at the time they get the task.  Clients generally need to be able to do this sort of verification in a few places (e.g. also after reconnect, to decide whether to re-volunteer or not).

I think this will still be a best practice even if/when we do have a "complete" op, because a disconnected client is not guaranteed to see the "complete" on reconnect (i.e. if a summary happens after the "complete" but before their reconnect).